web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx

```diff
@@ -119,22 +119,10 @@ const properties: ExternalCallFilterProperties = {
     visualToggle: {
       '0': {
         show: [],
-        hide: [
-          'scheduleIds',
-          'outOfScheduleTargetType',
-          'outOfScheduleLocution',
-          'outOfScheduleNumberCountry',
-          'outOfScheduleNumberValue',
-        ],
+        hide: ['scheduleIds', 'outOfScheduleTargetType'],
```

Comment:
outOfScheduleLocution must also be hidden/shown if  outOfScheduleEnabled is not disabled/enabled, like the locution for hollidayEnabled

```
  holidayEnabled: {
    label: _('Holidays enabled'),
    enum: {
      '0': _('No'),
      '1': _('Yes'),
    },
    default: '0',
    visualToggle: {
      '0': {
        show: [],
        hide: ['calendarIds', 'holidayTargetType', 'holidayLocution'],
      },
      '1': {
        show: ['calendarIds', 'holidayTargetType', 'holidayLocution'],
        hide: [],
      },
    },
  },